macaroonbakery/codec.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+import base64
+import json
+import six
+from nacl.public import Box, PublicKey
+from nacl.encoding import Base64Encoder
+
+import bakery
+import macaroon
+
+
+def encode_caveat(condition, root_key, third_party_info, key, ns):
+    '''Encrypt a third-party caveat with the given condtion
```

Comment:
Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.  See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings